Tycho r2.0.1+

yhetti opened the pull request.

yhetti committed:
Add more measurements to Drawing laye; Fix TPS55288 footprint

yhetti committed:
Cahnge TC7USB42 to PI3USB102GZLEX and update footprint to match Cynthion

yhetti committed:
Update PCB footprints; update date/rev

martinling reviewed (changes requested):
With the new footprint for U4, pins 1, 2, 4, 5 and 9 now have tracks running onto them that are wider than the pads. These tracks should be reduced so that the fabricated pads consistently have the width defined in the footprint.

![image](https://github.com/greatscottgadgets/tycho/assets/673823/9e31f09f-7a33-41b1-9d98-2d644f231001)

For the same reasons, I think it would be better to remove the GND polygon around pin 3, and run a track into this pin instead. When trying to rework this part, the extra heat capacity of the GND net was a significant issue.

On Cynthion we do have a polygon around pin 3, but we also have a polygon around pin 8 which serves to balance it out on the opposite corner. On Tycho we don't have this option, as pin 8 is in use to disable the switch, rather than being permanently grounded.

On the User.Drawings layer, the 92mm dimension on the right is no longer aligned with the holes. Its value is correct but it has been shifted upward slightly. The new dimensions look OK. Should there perhaps be one for the 24V socket too?

I think that the TPS55288 footprint is still not quite correct. The top edges of pads 1 and 19 do not line up with the top edges of pads 20-23.

mossmann commented on tycho/tycho.kicad_pro:
```diff
@@ -648,7 +648,7 @@
-    "DATE": "2024-03-07",
-    "VERSION": "r2.0.1"
+    "DATE": "2024-04-09",
+    "VERSION": "r2.0.1+"
```
This is fine, but in the future please try to remember to update the version with a "+" before making any other changes.

mossmann commented on tycho/tycho.kicad_pcb:
```diff
@@ -4417,7 +4417,7 @@
       (tstamp 9c904c40-d6ca-41d1-8f30-fe11b7be348a)
     )
-    (fp_text value "42.2k" (at 2.1 0 -180) (layer "F.Fab")
+    (fp_text value "22k" (at 2.1 0 -180) (layer "F.Fab")
```
The change to the R65 value is layout/schematic parity fix for r2.0.1. This is fine, but in the future try to make such changes first before updating the version to "+".

mossmann commented on tycho/tycho.kicad_pcb:
```diff
@@ -8215,83 +8215,83 @@
-      (net 2 "VCC") (pinfunction "VCC") (pintype "power_in") (tstamp b1d4cbe1-7ff0-4d00-8ed3-fd844db4a8bb))
-    (pad "10" smd rect (at -0.4 -0.7685 180) (size 0.225 0.563) (layers "F.Cu" "F.Paste" "F.Mask")
-      (net 22 "/D_S_1") (pinfunction "S") (pintype "input") (tstamp f720bc90-c600-4461-888c-a9843fa094a7))
+    (fp_text value "PI3USB102G" (at 0 2 180) (layer "F.Fab")
```
Please hide the U4 Value on F.Fab.

mossmann commented on tycho/tycho.kicad_pcb:
```diff
@@ -8215,83 +8215,83 @@
-    (fp_text value "~" (at 1.5 0 90) (layer "F.Fab") hide
-        (effects (font (size 0.5 0.5) (thickness 0.1)))
-      (tstamp df6c8133-8cc5-42e8-b0c1-8d89c794ad9f)
+    (fp_text reference "U4" (at 0 -2 180) (layer "F.SilkS")
```
Please fix the "U4" silkscreen conflict on F.SilkS.

mossmann reviewed (changes requested):
The U6 footprint is much improved, but I would also like to see silkscreen, fab, and courtyard layers added to it. I've requested some other small changes as well.

mossmann commented:
This isn't specific to this PR in particular, but I would like .kicad_prl files removed from the repo and added to .gitignore.

yhetti committed:
Update TPS55288 footprint

	*fix pad size of pads 1 & 19
	*add pin 1 marking to silkscreen
	*add component outline and refdes to fab layer
	*add courtyard outline

yhetti committed:
Fix U4 track widths; remove zone from around U4 GND pad

add *.kicad_prl to .gitignore

yhetti committed:
Remove .kicad_prl files; update date

mossmann reviewed (changes requested):
The U6 footprint updates look good.

Only one end of the 92 mm dimension was corrected, making the dimension value incorrect.

The trace widths escaping from U4 are generally improved, but I would like to see pins 1 and 2 routed nicely like pins 4 and 5, avoiding the excess pin 1 pad width.

Fix the clearance violations introduced by the changes to `D_SW1+` and `D_SW1-` routing. These can be fixed by refilling zones.

Always run DRC and refill zones before creating or updating a pull request.

Remove the zone around U4 pin 3. A commit message says this was done, but the zone is still there.

Restore the `HOST_VBUS` zone that became unfilled. This can be fixed by refilling zones.

Add a dimension for the 24 V socket (J8) as @martinling suggested.

yhetti committed:
Remove pad from silkscreen layer in TESTPROBE footprint and update pcb

yhetti committed:
Resolve comments on PR#14

	*fix 92 mm dimension
	*move trace on pad 1 of U4
	*re-fill zones
	*remove zone around pin 3 of U4
	*add dimension for J8

yhetti committed:
Update date

martinling commented:
The dedicated polygon around U4 pin 3 was removed, but the main GND polygon for the top layer still fills some of the area around this pin. To get a clean track running into that pin it's necessary to add a keepout area.

![image](https://github.com/greatscottgadgets/tycho/assets/673823/087c4e7a-eeee-4c2e-bdd8-34bdd800c610)
![image](https://github.com/greatscottgadgets/tycho/assets/673823/a61261a2-fd7b-471a-881f-c5b26aa035ba)

I have this change in 633c8e664edbbf24113529f95da32f2e54311757 if you want it.

mossmann reviewed (changes requested):
The updates look good, but please add a keepout to U4 pin 3 as @martinling suggested.

yhetti committed:
Add a keepout area around U4 pin 3

yhetti committed:
Update date

mossmann reviewed (approved):
Looks good. Thank you!
